### DISCUSSION TITLE
[Question] How to override simulation app experience kit after simulation app is constructed

### DISCUSSION THREAD

**Discussion by zoctipus** [isaac-sim/IsaacLab#1124] [Q&A]
### Question

I am trying to write a scheduler that automate the training and evaluation(with recording videos), the terminal argument such as enable camera are not needed for training, but needed for evaluation. However scheduler is only constructed after simulation app is available, if scheduler says next job is eval, I want to override simulation app with no camera_enabled, if the scheduler says next job is train, I want to override simulation app headless and camera_enabled. May I wonder how to approach it?

For now I can keep enable_camera to be true while set video = True for eval and set video = False for train

**Comment by kellyguo11** [comment]
Hello, it is not possible to switch to a different app file once the simulation app has been initialized. In this case, using the app file with cameras enabled sounds like the right approach. However, there are some settings from the app files that can be modified at runtime through carb.settings.
